common/persistence/visibility/factory.go

```diff
@@ -138,7 +179,7 @@ func NewStandardManager(
 }
 
 func NewAdvancedManager(
-	defaultIndexName string,
+	indexName string,
```

Comment:
The reason is was called `defaultIndexName` because there is partial support for index per namespace. Metadata has `indexName` field which is not propagated and used yet. The one from config supposed to be default index name if it is not configured on namespace level. I think you should rename it back.

Reply:
reverted